Cargo.toml

```diff
@@ -199,7 +200,7 @@ mem_forget = "deny"
 multiple_crate_versions = "allow"
 or_fun_call = "deny"
 panic = "deny"
-panic_in_result_fn = "deny"
+# panic_in_result_fn = "deny" -- we cannot disable this for tests to use assertions
```

Comment:
it also complains about assertions... I'm fine with removing this one now. In general, I think we're now reasonably good about using assert/panic for violations of invariants (programmer error) rather than routine errors